lib/hackmode/p2p-backend-hackmode-peer.test.js

```diff
@@ -1,5 +1,5 @@
 const { $sleep, unhandledRejection } = require('../n3h-common')
-unhandledRejection.strict()
```

Comment:
I absolutely cannot figure out how to catch the uncaught ones and they're very problematic

Reply:
put a logging catch on the connect tracking... still not a great solution, but works for now